test/e2e/rke2/variables.tf

```
@@ -0,0 +1,52 @@
+variable "name" {
+  description = "Name for cluster"
+  type        = string
+  default     = "dubbd-rke2-ci"
```

```suggestion
  default     = "packer-rke2-ci"
```
Missed this one earlier - would be good to name accordingly to help track CI resources.